app/models/assessment.rb

```diff
@@ -12,6 +12,9 @@ class Assessment < ApplicationRecord
 
   enum visibility: %i[internal external]
 
+  # serializer stores values here temporarily before they're sent to metadata
+  attr_accessor :metadata
```

Comment:
also stored here when read back from metadata repo?

Reply:
yep, good point. I'll update the comment to be accurate